packages/rest-client/src/generators/openapi-client/generator.spec.ts

```diff
@@ -21,6 +38,18 @@ describe('openapi-client generator', () => {
         await generator(tree, options);
         const config = readProjectConfiguration(tree, 'test-client');
         expect(config).toBeDefined();
+
+        snapshotFiles(tree, [
+            joinPathFragments('test-client', 'project.json'),
+            joinPathFragments('test-client', 'tsconfig.json'),
+            joinPathFragments('test-client', 'tsconfig.lib.json'),
+            joinPathFragments('test-client', 'tsconfig.spec.json'),
+            joinPathFragments('test-client', '.eslintrc.json'),
+            joinPathFragments('test-client', 'package.json'),
+            joinPathFragments('test-client', 'jest.config.ts'),
+            joinPathFragments('test-client/src', 'index.ts'),
+            joinPathFragments('test-client', 'README.md'),
```

Comment:
Do we have coverage in e2e for the orval generator? Checking that orval actually runs under the hood and builds our endpoint? Do we then build/test that generated client?